utilities/src/main/java/org/craftercms/commons/config/profiles/AbstractProfileConfigMapper.java

```diff
@@ -54,7 +54,7 @@ public T readConfig(ConfigurationProvider provider, String module, String path,
                 .stream()
                 .filter(c -> profileId.equals(c.getString(CONFIG_KEY_ID)))
                 .findFirst()
-                .orElseThrow(() -> new ConfigurationException("Profile '" + profileId + "' not found"));
+                .orElseThrow(() -> new ConfigurationProfileNotFoundException("Profile '" + profileId + "' not found"));
```

Comment:
Let's use String.format instead of concatenation

Reply:
Fixed to use String.format